clang/lib/Basic/SourceManager.cpp

```diff
@@ -350,6 +350,7 @@ void SourceManager::clearIDTables() {
   LastLineNoContentCache = nullptr;
   LastFileIDLookup = FileID();
 
+  IncludedLocMap.clear();
```

Comment:
Is there a way to write a unit test checking this behavior?

SourceManager should be relatively easy to mock, I think it'd be useful to write one.

Reply:
done